include/sliceDataStorage.h

```diff
@@ -212,6 +212,8 @@ class SupportLayer
     std::vector<SupportInfillPart> support_infill_parts; //!< a list of support infill parts
     Polygons support_bottom; //!< Piece of support below the support and above the model. This must not overlap with any of the support_infill_parts or support_roof.
     Polygons support_roof; //!< Piece of support above the support and below the model. This must not overlap with any of the support_infill_parts or support_bottom.
+    Polygons support_fractional_roof_top; //!< If the support distance is less than a multiple of the layer height,
```

Comment:
```suggestion
    Polygons support_fractional_roof_top; //!< If the support distance is not exactly a multiple of the layer height,
```
Bit nitpicky, but the support distance is always less then a multiple of a layer height

```
support distance < layer height * n
```
There always exists an `n` for which the support distance is less the `n * layer height`